quickwit/quickwit-query/src/elastic_query_dsl/term_query.rs

```diff
@@ -54,7 +55,9 @@ enum TermValue {
 }
 
 fn deserialize_term_value<'de, D>(deserializer: D) -> Result<String, D::Error>
-where D: Deserializer<'de> {
+where
+    D: Deserializer<'de>,
+{
```

Comment:
the CI disagree with this change, you might not be using the same rustfmt as it does